app/models/authorization.rb

```diff
@@ -0,0 +1,7 @@
+class Authorization < ApplicationRecord
+  # include Encryptable
+  # attr_encrypted :token, :secret, :refresh_token
+
+  belongs_to :admin, optional: true
+  validates_uniqueness_of :uid, scope: [:provider]

```

Comment:
use, please, this syntax https://guides.rubyonrails.org/active_record_validations.html#uniqueness